src/renderer/components/ft-icon-button/ft-icon-button.vue

```diff
@@ -1,28 +1,48 @@
 <template>
   <div class="ftIconButton">
-    <font-awesome-icon
-      ref="iconButton"
-      class="iconButton"
-      :title="title"
-      :icon="icon"
-      :class="{
-        [theme]: true,
-        shadow: useShadow,
-        disabled
-      }"
+    <font-awesome-layers
       :style="{
-        padding: padding + 'px',
-        fontSize: size + 'px'
+        width: `${size + padding * 2}px`,
+        height: `${size + padding * 2}px`,
       }"
-      tabindex="0"
-      role="button"
-      :aria-disabled="disabled"
-      :aria-expanded="dropdownShown"
-      @click="handleIconClick"
-      @mousedown="handleIconMouseDown"
-      @keydown.enter.prevent="handleIconClick"
-      @keydown.space.prevent="handleIconClick"
-    />
+    >
+      <font-awesome-icon
+        ref="iconButton"
+        class="iconButton"
+        :title="title"
+        :icon="icon"
+        :class="{
+          [theme]: true,
+          shadow: useShadow,
+          disabled
+        }"
+        :style="{
+          padding: padding + 'px',
+          fontSize: size + 'px'
+        }"
+        tabindex="0"
+        role="button"
+        :aria-disabled="disabled"
+        :aria-expanded="dropdownShown"
+        @click="handleIconClick"
+        @mousedown="handleIconMouseDown"
+        @keydown.enter.prevent="handleIconClick"
+        @keydown.space.prevent="handleIconClick"
+      />
+      <font-awesome-layers-text
+        v-if="counterValue > 0"
+        class="iconCounter"
+        :class="{
+          [theme]: true,
+        }"
+        :style="{
+          fontSize: `${size + padding * 2}px`,
+        }"
+        counter
+        :value="counterValue"
```

Comment:
**issue (blocking):** There's currently no non-visual or semantic indication of the meaning, which makes it inaccessible to screen reader users, and somewhat confusing for people who don't intuitively grasp the meaning.

Reply:
Would updating the title make it better? (good enough or not is something else)
![image](https://github.com/FreeTubeApp/FreeTube/assets/1018543/5c724e81-0b1e-4fbd-ae48-6f1d31fcb27d)

Reply:
I think that's a safe way to do it. That label seems pretty good too, albeit maybe a bit too long. Maybe `Add to Playlist (Aleady Added to 2)`?

Reply:
Updated